src/arch/amdgpu/vega/gpu_registers.hh

```diff
@@ -129,7 +129,7 @@ namespace VegaISA
         REG_LDS_DIRECT = 254,
         REG_SRC_LITERAL = 255,
         REG_VGPR_MIN = 256,
-        REG_VGPR_MAX = 511
+        REG_VGPR_MAX = 767
```

Comment:
Ideally it would be good to know where these magic numbers come from

Reply:
I'm wondering about this too since Table 22 in the mi300 specs says the VGPR value is from 256-511, and `REG_VGPR_MIN` is 256 but `REG_VGPR_MAX` is not 767 instead of 511

Reply:
Dropped this commit.  It seems there isn't an index for AV0-AV255 so this change might not be correct.

Reply:
ok, I will approve again then.  Thanks @mjkpolo for spotting this!